astro/src/content/docs/operate/roadmap/deprecation-policy.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Deprecation Policy
+description: Learn more about the FusionAuth deprecation policy.
+navcategory: admin
+section: operate
+subcategory: roadmap
+---
+
+## Overview
+
+FusionAuth is an architectural component integrated into your applications or systems. Such integration means that when changes to FusionAuth occur, it can negatively affect your application. As developers ourselves, we strive to maintain backwards compatibility. 
+
+However, there are times when features or functionality must be removed. FusionAuth has [extensive release notes](/docs/release-notes/), which are the source of truth for changes to the FusionAuth system, and any deprecated features will be listed there.
```

Comment:
Can we mention deprecation and changes? We often document changes that we don't necessary consider breaking in a general sense, but may change the expected behavior for some use cases. 

> and any deprecated features or changes to existing features will be listed there